ddtrace/compat.py

```diff
@@ -1,15 +1,39 @@
-import platform
-import random
-import re
-import sys
-import textwrap
-import threading
-from typing import Any
-from typing import AnyStr
-from typing import Text
-from typing import Union
-
-import six
+from ddtrace.utils.deprecation import deprecation
+
+from .internal.compat import CONTEXTVARS_IS_AVAILABLE  # noqa: F401
```

Comment:
you can probably get around `F401` by defining `__all__ = ('CONTEXTVARS_IS_AVAILABLE', ...)`